src/js/utils/cursor/position.js

```diff
@@ -63,6 +63,10 @@ const Position = class Position {
     return this.markerPosition.offset;
   }
 
+  get isBlank() {
+    return false;
```

Comment:
No particular reason; I can change it. FWIW I don't think that defining a getter using `get X()` versus setting it on `this` in the constructor changes the shape of the object, in case that is a concern. I'll put it in the constructor for clarity, though.